terraform/modules/rds/variables.tf

```diff
@@ -25,7 +25,7 @@ variable "rds_db_engine" {
 }
 
 variable "rds_db_engine_version" {
-  default = "12.8"
+  default = "12.11"
```

Comment:
Yep, I think that's correct. All of our databases will upgrade, but customer databases are configured separately and will be unaffected.

Reply:
It would if they are still using the shared RDS instance, that is deprecated for a while now. For knowledge, shared RDS was provisioned by TF then broker made an user so they could access it.